.github/actions/release/action.yaml

```diff
@@ -140,13 +140,14 @@ inputs:
           os: linux
           arch: x86_64
       ```
-  github-tag-template:
+  github-tag-templates:
     default: '${version}'
     type: string
     description: |
-      GitHub tag template to use for the created release tag. Currently only supported template-var:
+      Comma-separated list of GitHub tag templates to use for the created release tags. Each template can use
       ${version} (bash-syntax): Version read from local component descriptor (i.e. previous version
       operations are honoured).
+      The first tag in the list will be used to create the GitHub release.
```

Comment:
how about defining an additional input:
- keep github-tag-template
- add: git-extra-tag-templates (optional)

This makes will make the contract more obvious, even w/o reading the documentation

Reply:
I felt it would be more natural defining all the tags in one single list, rather than splitting into a single template and another list of templates